<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientConnectionGui</class>
 <widget class="QWidget" name="ClientConnectionGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_connectSerialClient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_serialClient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>561</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>231</width>
     <height>316</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_serialports"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ports</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_baudrate"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_databits"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_parity"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_stopbits"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_flowcontrol"/>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_connectionTimeout"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Baudrate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Databits</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Stopbits</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Flowcontrol</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Connection timeout</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_readTimeout"/>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spinBox_writeTimeout"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Read timeout</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Write timeout</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>linebreak</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="comboBox_linebreak"/>
    </item>
    <item row="10" column="1">
     <widget class="QRadioButton" name="radioButton_serial_connected">
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Write data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_writeDataSerialClient"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_msSleepSerialClient"/>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_serialWriteMultiple">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Serial Client&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Socket Client&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>231</width>
     <height>279</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Connection timeout</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_connectionTimeoutSocketClient"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Read timeout</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_readTimeoutSocketClient"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Write timeout</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_writeTimeoutSocketClient"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>linebreak</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="comboBox_linebreakSocketClient"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ipSocketClient"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_portSocketClient"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Reconnect timeout</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_reconnectTimeoutSocketClient">
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
      <property name="value">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Reconnect </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="radioButton_reconnect_on_timeout">
      <property name="text">
       <string>Reconnect</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QRadioButton" name="radioButton_socket_connected">
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_connectSocketClient">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Write data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_writeDataSocketClient"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_msSleepSocketClient"/>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_socketWriteMultiple">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_disconnectSerialClient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_disconnectSocketClient">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
